src/main/java/step2/StringParser.java

```diff
@@ -0,0 +1,23 @@
+package step2;
+
+import java.util.Objects;
+
+public class StringParser {
+    private final String value;
+
+    public StringParser(String value) {
+        validate(value);
+
+        this.value = value;
+    }
+
+    private void validate(String value) {
+
+        if (Objects.isNull(value) || value.isBlank())
+            throw new IllegalArgumentException("입력 값이 null 이거나 빈 공백 문자입니다");
```

Comment:
```suggestion
        if (Objects.isNull(value) || value.isBlank()) {
            throw new IllegalArgumentException("입력 값이 null 이거나 빈 공백 문자입니다");
        }
```

자바의 경우 한 줄의 if문에도 괄호를 사용해주는 것이 컨벤션입니다!

https://google.github.io/styleguide/javaguide.html 또는 https://myeonguni.tistory.com/1596를 참고해주세요 :)